pkg/container/bubblewrap_runner.go

```diff
@@ -34,7 +34,10 @@ import (
 
 var _ Debugger = (*bubblewrap)(nil)
 
-const BubblewrapName = "bubblewrap"
+const (
+	BubblewrapName = "bubblewrap"
+	BuildUserID    = "1000"
```

Comment:
I wouldn't export this, since it's not used anywhere outside this package, and `container.BuildUserID` might be misleading to external readers of this package's API.

Reply:
Good catch! Thanks @imjasonh. Addressed, PTAL